### PR TITLE
Rename walking_transfer_penalty and add walking_transfer_penalty

### PR DESCRIPTION
**Pull request by kadhikari**
* Modify existing parameter walking_transfer_penalty to arrival_transfer_penalty
* Add a new parameter walking_transfer_penalty in jormun, database tyr as well as in api tyr
* Modify kraken to use this parameter.

**Note:**
* No test added on existing parameter walking_transfer_penalty modified to arrival_transfer_penalty
* A test added on newly added parameter walking_transfer_penalty only for distributed as new_default will be deleted soon.

Ticket: https://jira.kisio.org/browse/NAVP-1667

### CODE REVIEW

File: source/kraken/worker.cpp [hunk 1]
```diff
@@ -780,8 +780,9 @@ void Worker::journeys(const pbnavitia::JourneysRequest& request,
                 routing::make_response(
                     this->pb_creator, *planner, arg.origins[0], arg.destinations[0], arg.datetimes, request.clockwise(),
                     arg.accessibilite_params, arg.forbidden, arg.allowed, *street_network_worker, arg.rt_level,
-                    seconds{request.walking_transfer_penalty()}, request.max_duration(), request.max_transfers(),
-                    max_extra_second_pass, request.free_radius_from(), request.free_radius_to(),
+                    seconds{request.arrival_transfer_penalty()}, seconds{request.arrival_transfer_penalty()},
```
Comment (pbench) [inline source/kraken/worker.cpp]:
```suggestion
                    seconds{request.arrival_transfer_penalty()}, seconds{request.walking_transfer_penalty()},
```

**Review by pbench — CHANGES_REQUESTED** [review]
A small typo, otherwise everything looks good ! Thanks a lot :+1:

**Review by pbench — APPROVED** [review]

File: source/jormungandr/tests/journey_common_tests.py [hunk 2]
```diff
@@ -242,6 +242,31 @@ def test_datetime_represents_arrival(self):
...
+        assert len(response["journeys"]) == 3
+
+        response = self.query_region(query + '&_walking_transfer_penalty=120')
+        assert len(response["journeys"]) == 3
+
+        response = self.query_region(query)
+        assert len(response["journeys"]) == 3
+
+        response = self.query_region(query + '&_walking_transfer_penalty=480')
+        assert len(response["journeys"]) == 2
```
Comment (pbench) [inline source/jormungandr/tests/journey_common_tests.py:268]: great !

File: source/jormungandr/tests/routing_tests_new_default.py [hunk 3]
```diff
@@ -327,6 +327,12 @@ def test_same_origin_and_destination_point_in_request(self):
         assert response[1] == 400
         assert "your origin and destination points are the same" in response[0]["message"]
 
+    def test_walking_transfer_penalty_parameter(self):
+        """
+        This feature is no longer supported
+        """
+        pass
+
```
Comment (pbench) [inline source/jormungandr/tests/routing_tests_new_default.py:335]: Why do you add this test ?

**Review by pbench — APPROVED** [review]

File: source/jormungandr/tests/routing_tests_new_default.py (hunk 3 above)
Comment (kadhikari) [reply source/jormungandr/tests/routing_tests_new_default.py:335]: As i explained above the test in new_default doesn't work and this scenario will certainly disappear in future.

Comment (pbench) [reply source/jormungandr/tests/routing_tests_new_default.py:335]: but you have to add it to `routing_tests_new_default.py` ?

Comment (kadhikari) [reply source/jormungandr/tests/routing_tests_new_default.py:335]: Yes

**Review by azime — APPROVED** [review]

**Pull request merged**